### DISCUSSION TITLE
Automatically select a specific auth strategy

### DISCUSSION THREAD

**Discussion by lfernandes00** [requarks/wiki#6985] [Help / Questions] [answered]
I have a wiki js app with 2 Azure AD auth strategies (strat1 & strat2) both in the /login directory that shows me the 2 strategies that I can choose to authenticate.

The second strategy (strat2) is an Azure AD B2C authentication that I have in common with a next js app, that has a button that redirects me to the wiki js app.

I want to know if its possible to redirect directly to the strat2 in order to have a SSO between the 2 apps, instead of being redirected to the login and having to choose the strat2.

Is this possible? How can I achieve it?

**Comment by NGPixel** [comment, marked as answer]
Use `/login/<strategy-uuid>` where `<strategy-uuid>` is the id of the strategy as found in the admin area.

Also next time, don't spam your question everywhere...

**Reply by lfernandes00** [reply]
Thank you for the quick response! And sorry about the spam..
